NewsApp/API/NewsAPIEnvironmentPath.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  APIEnvironmentPath.swift
+//  Feshar
+//
+//  Created by Abdelrhman Eliwa on 4/7/20.
+//  Copyright © 2020 Abdelrhman Eliwa. All rights reserved.
+//
+
+import Foundation
+
+enum NewsAPIService {
+    case headlines
+    case everything
+    
+    func baseURL() -> String {
+        switch self {
+        case .headlines:
+            return
+                NewsAPIEnvironmentPath.development.scheme() +
+                NewsAPIEnvironmentPath.development.host() + "/v2/" + "top-headlines?"
+        case .everything:
+            return
+                NewsAPIEnvironmentPath.development.scheme() +
+                NewsAPIEnvironmentPath.development.host() + "/v2/" + "everything?"
+        }
+    }
```

Comment:
Base url should be added in a private property/ method to avoid duplication in 
```
NewsAPIEnvironmentPath.development.scheme() +
                NewsAPIEnvironmentPath.development.host() + "/v2/"
```
Consider return the value as URL